### DISCUSSION TITLE
Data not syncing anymore when upgrading @electric-sql/react to 0.2 or higher

### DISCUSSION THREAD

**Discussion by Makisuo** [electric-sql/electric#1524] [Bug reports]
As said in the title when upgrading, data will always be undefined, even though the response in the network tab seems fine.

Here is  a example response of the shape I'm trying to sync, no errors are thrown.
`[
    {
        "offset": "27003656_0",
        "value": {
            "created_at": "2024-08-14 20:06:16.040503+00",
            "deleted_at": null,
            "id": "1",
            "name": "wow",
            "provider_id": "github",
            "resources": "{users,repos}",
            "updated_at": "2024-08-14 20:06:16.040503+00"
        },
        "key": "\"public\".\"collections\"/\"1\"",
        "headers": {
            "relation": [
                "public",
                "collections"
            ],
            "action": "insert",
            "txid": 753
        }
    },
    {
        "headers": {
            "control": "up-to-date"
        }
    }
]`

**Comment by KyleAMathews** [comment]
Hey it looks like you need to pull the latest docker image as there was a breaking change for the server.

`docker pull electricsql/electric`

**Reply by Makisuo** [reply]
Ah missed the memo of the docker container getting moved back from electric-next, thanks!
